### ISSUE TITLE
Rebasing from fedora silverblue 38 result in no nautilus nor gnome software center

### ISSUE THREAD

**Issue by Froggy232** [ublue-os/main#144]
Hi,
I try to switch from fedora silverblue 38 to Ublue OS Main 38.
Problem is : when I try to rebase to it, it remove a bunch of things including nautilus and gnome-software-center.
I have tried to type "rpm-ostree reset" after rebasing to it, but it removed even more things and the computer didn't boot anymore, I needed to rollback to an older deployment.
Does someone have an idea?
Thanks you

**Comment by castrojo** [comment]
Can you paste in text of what it wants to remove? And also an output of `rpm-ostree status -v` would be useful, thanks!

**Comment by Froggy232** [comment]
Thanks for your quick answer!
So, the output of the changes of the rebase is here : https://pastebin.com/RfhuEjBg
The output of "rpm-ostree status -v" is here : https://pastebin.com/ZW2CMDbD
Thanks you!

**Comment by castrojo** [comment]
Yes, you seem to be trying to move to one of our images while having existing layers. I recommend doing the `rpm-ostree reset` command _before_ trying to rebase. 

I would also remove `copr:copr.fedorainfracloud.org:hyperreal:better_fonts` `copr:copr.fedorainfracloud.org:zawertun:hack-fonts` so that they don't conflict with everything.

Once your silverblue install is back to it's stock, reset configuration the rebase should work fine. Then after that is finished you layer additional things at that step, though as you've discovered, layering can have some complexity issues.

It's nice that you have working pin to rollback too, nice work there. 😄

**Comment by Froggy232** [comment]
Thanks, I will try that and tell you, but the thing is I have already try to reset before doing the upgrade, and it wasn't changing anything.
I will try to remove the repo.
Thanks you!

**Comment by castrojo** [comment]
Yeah if you can do that, then reboot, then paste in the `rpm-ostree status -v` it'll be easier to tell. I run a status after each reboot to see where I am in the process. It might be those third party repos too, hard to tell unless you went and figured out the dependencies for the packages they provide.

**Comment by joshua-stone** [comment]
You rebased onto `base-main` which lacks a desktop environment. You should rebase `silverblue-main` instead:

```
$ rpm-ostree rebase ostree-unverified-registry:ghcr.io/ublue-os/silverblue-main:38
```

**Comment by Froggy232** [comment]
Oh, sorry, it explain everything, but isn't silverblue-main deprecated?
I had understand that, but maybe I'm confusing somethings.
Thanks you

**Comment by joshua-stone** [comment]
The [`ublue-os/base`](https://github.com/ublue-os/base) repo has been deprecated in favor [`ublue-os/main`](https://github.com/ublue-os/main), which provides various flavors with `silverblue-main` being one of them:

https://github.com/orgs/ublue-os/packages?repo_name=main

**Comment by Froggy232** [comment]
I understand now, I tried your command and everything work.
Thanks you!

**Comment by castrojo** [comment]
I've removed all the "base" images from the website so hopefully it means people won't stumble across this image unless they're looking for it specifically.
